services/issue/label.go

```diff
@@ -84,7 +36,7 @@ func AddLabels(issue *models.Issue, doer *models.User, labels []*models.Label) e
 		return err
 	}
 
-	sendLabelUpdatedWebhook(issue, doer)
+	notification.NotifyIssueChangeLabels(doer, issue, nil, nil)
```

Comment:
```suggestion
	notification.NotifyIssueChangeLabels(doer, issue, labels, nil)
```
why not use the new parameter - we can implement the handling at NotifyIssueChangeLabels later?